src/app/containers/MostRead/utilities/processMostRead.js

```diff
@@ -60,48 +57,44 @@ const mostReadItems = ({ data, isAmp, numberOfItems, service }) => {
   if (!data) {
     return null;
   }
-  const records = pathOr([], ['records'], data);
 
-  // The ARES test endpoint for most read renders fixture data, so the data is stale
-  const isTest = process.env.SIMORGH_APP_ENV === 'test';
+  const filteredData = filterPopularStaleData({
+    data,
+    isAmp,
+    service,
+    popularType: 'mostRead',
+  });
+
+  if (!filteredData) {
+    return null;
```

Comment:
Is it worth adding some logging here - if there is not already logging within `filterPopularStaleData`?

Reply:
Logging is already done at `filterPopularStaleData`